README.md

```diff
@@ -1,24 +1,42 @@
 # Northcoders News API
 
-The purpose of this API is to access application data programmatically. This is mimicing the building of a real world backend service (similar to reddit).
+The purpose of this API is to access application data programmatically. This is mimicing the building of a real world backend service (similar to reddit). NC News contains articles containing a body, author, topic, votes, and comment count, along with the date it was created.
 
-To set up the database correctly you'll need the following commands:
+Available Endpoints:
+GET /api - serves all the available endpoints of the api
+GET /api/topics - serves an array of all topics
+GET /api/articles - serves an array of all articles
+GET /api/articles/:article_id - serves an object containing the requested article
+PATCH /api/articles/:article_id - Updates content on the requested article
+GET /api/articles/:arrticle_id/comments - serves an array of all comments associated with the requested article
+POST /api/articles/:article_id/comments - Adds a new comment to the requested article
+DELETE /api/comments/comment_id - Deletes the comment by the comment id
+GET /api/users - serves an array of all users
 
-npm run...
-setup-dbs - to set up the database
-seed - to seed the database
-test - to run the testing suite
-prepare - to install Husky
+The technology stack used:
+Node.js
```

Comment:
think you can bullet point these in markdown with - 

Have a look at markdown syntax to make this readme look even better!

Similarly surrounding any code or terminal commands with 3 backticks will make a code block or single backticks for any inline code :)